go.mod

```diff
@@ -54,3 +54,5 @@ require (
 	k8s.io/client-go v0.21.3
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v0.21.8-0.20211221144726-c4e27e8c9c78
```

Comment:
do we need replace ? can we merge the IDL changes first

Reply:
yes waiting on flyteidl merge. This was needed for quick try out